.circleci/config.yml

```diff
@@ -427,7 +437,10 @@ commands:
             wget --output-document /dev/null --waitretry=5 --tries=10 --retry-connrefused localhost:9200 || true
       - run:
           name: run integration test
-          command: mvn -B org.jacoco:jacoco-maven-plugin:report org.jacoco:jacoco-maven-plugin:report-aggregate clean install -P$TESTING_PROFILE,coverage -ntp
+          command: |
+            # Adding all "normal" certs into this local one that has the Hoverfly cert (instead of adding Hoverfly cert to the global one so it doesn't potentially affect other tests)
```

Comment:
for my understanding, does this comment make sense anymore? due to the `TESTING_PROFILE` variable, this means pretty much all tests minus unit tests are affected?

Reply:
Hmm, I guess "affect other tests" doesn't really apply here since only one specific testing profile is running per job. Will update comment